linera-views/src/scylla_db.rs

```diff
@@ -15,6 +15,9 @@
 //! [trait1]: common::KeyValueStore
 //! [trait2]: common::Context
 
+/// Fundamental constant in ScyllaDB: The maximum size of a multi keys query
```

Comment:
Do you have a link to the documentation? I couldn't find it anywhere. Is it the default value for `--max-clustering-key-restrictions-per-query`?

Reply:
I could not find it either in the `scylla.yaml` nor in the CQL documentation. I will merge it and look at the scyllaDB source code.